mantle/kola/harness.go

```diff
@@ -665,6 +665,12 @@ ExecStart=%s
 `, KoletExtTestUnit, kolaExtBinDataEnv, kolaExtBinDataDir, remotepath)
 	config.AddSystemdUnit(KoletExtTestUnit, unit, conf.NoState)
 
+	// Architectures using 64k pages use slightly more memory, ask for more than requested
+	// to make sure that we don't run out of it. Currently only ppc64le uses 64k pages.
+	if system.RpmArch() == "ppc64le" && targetMeta.MinMemory < 8192 {
+		targetMeta.MinMemory = 8192
+	}
```

Comment:
How about we instead do `* 2`? That way, tests which truly don't need lots of memory aren't unnecessarily constricting how many can run in parallel.

Reply:
Since `targetmeta.MinMemory` is defined in JSON, doing `* 2` could mean that we end up with up to 16Gb of RAM? 

IMO, having a platform default is a bit better. 

I dunno, going with a floor of 8Gb is _safter_ than a blind *2`

Reply:
I'm mostly concerned with allocating way more memory than necessary for trivial tests like `misc-ro`. Something like this would work

```go
if system.RpmArch() == "ppc64le" && targetMeta.MinMemory < 4096 {
    targetMeta.MinMemory = targetMeta.MinMemory * 2
}
```

to cap the "auto-scaling" at 8G (so in the worse case, it's the same thing this PR is doing, and in the best case it's much less hungry).

Reply:
Actually I have been thinking about it and it would probably make most sense to base it on the the (min) memory for current kola tests(now it is 1G for x86_64 and 2G for rest) and just do 4x for the ext tests(as currently it is basically that) and possibly specify the minmemory either via absolute, multiplier of the base(x times 1G/2G), but no less than 4/8G or the default 4/8G (or something like that). But it kind of sounded to me as bit of over-engineered, relatively big change to directly open/create PR for. What do you think?